website/package.json

```
@@ -3,22 +3,36 @@
   "version": "0.0.0",
   "private": true,
   "scripts": {
-    "start": "docusaurus-start",
-    "build": "docusaurus-build",
-    "gh-pages": "docusaurus-publish",
-    "examples": "docusaurus-examples",
-    "crowdin-upload": "crowdin --config ../crowdin.yaml upload sources --auto-update -b master",
-    "crowdin-download": "crowdin --config ../crowdin.yaml download -b master",
-    "publish-gh-pages": "docusaurus-publish",
-    "write-translations": "docusaurus-write-translations",
-    "version": "docusaurus-version",
-    "rename-version": "docusaurus-rename-version"
+    "start": "docusaurus start",
+    "build": "docusaurus build",
+    "swizzle": "docusaurus swizzle",
+    "deploy": "docusaurus deploy",
+    "serve": "docusaurus serve",
+    "crowdin:upload": "cd .. && crowdin upload sources --config crowdin-v2.yaml",
+    "crowdin:downloadTranslations": "cd .. && crowdin download --config crowdin-v2.yaml",
+    "write-translations": "docusaurus write-translations",
+    "fetchSupporters": "node fetchSupporters.js",
+    "netlify:ci:production": "yarn netlify:prepare && yarn netlify:crowdin && yarn build",
+    "netlify:ci:deployPreview": "yarn netlify:prepare && yarn build -l en",
+    "netlify:prepare": "yarn fetchSupporters && yarn build:js",
+    "netlify:crowdin": "yarn write-translations && yarn workspace @jest/monorepo crowdin:upload && yarn workspace @jest/monorepo crowdin:download"
```

```suggestion
    "netlify:crowdin": "yarn write-translations && yarn crowdin:upload && yarn crowdin:download"
```

All scripts with a colon (`:`) in their name are global so you don't need to specify the workspace here.

https://yarnpkg.com/cli/run/#details
https://yarnpkg.com/configuration/manifest/#scripts
https://yarnpkg.com/getting-started/qa/#how-to-share-scripts-between-workspaces

---

thanks 👍